framework/set/resources/providers/harvester/createResources.go

```diff
@@ -0,0 +1,50 @@
+package harvester
+
+import (
+	"os"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/rancher/tfp-automation/config"
+	"github.com/sirupsen/logrus"
+)
+
+// CreateHarvesterResources is a helper function that will create the Harvester resources needed for the RKE2 cluster.
+func CreateHarvesterResources(file *os.File, newFile *hclwrite.File, tfBlockBody, rootBody *hclwrite.Body, terraformConfig *config.TerraformConfig,
+	terratestConfig *config.TerratestConfig, instances []string) (*os.File, error) {
+	CreateTerraformProviderBlock(tfBlockBody)
+	rootBody.AppendNewline()
+
+	CreateHarvesterProviderBlock(rootBody, terraformConfig)
+	rootBody.AppendNewline()
+
+	CreateLocalBlock(rootBody, terraformConfig)
+	rootBody.AppendNewline()
+
+	for _, instance := range instances {
+		CreateHarvesterInstances(rootBody, terraformConfig, terratestConfig, instance)
+		rootBody.AppendNewline()
+	}
+
+	rootBody.AppendNewline()
+	dirList := file.Name()
+
+	localFile, err := os.Create(strings.Join(strings.Split(dirList, "main.tf"), "/") + "/local.yaml")
+	if err != nil {
+		logrus.Infof("Failed create local.yaml kubeconfig. Error: %v", err)
```

Comment:
These should be `logus.Errorf` so the test doesn't continue forward.

Reply:
shouldn't that be handled in the return?

Reply:
Ah, yeah you're right. All good here 👍🏿